content/topics/design/_index.md

```diff
@@ -6,52 +6,44 @@ slug: "design"
 
 # Weight
 weight: 2
 
-# display topic-collection template for this topic
-topic_collection: true
-
-# populate the topic-collection template with featured links
-data:
-  contents:
-    legislation:
-      name: "21st Century Integrated Digital Experience Act"
-      description: ""
-      link_text: "View policy"
-      image: "legislative-stamp-card-logo.png"
-      link_url: "/resources/21st-century-integrated-digital-experience-act/"
-  featured_resources:
-    multiple: false
-    resources:
-      - link: "/guides/hcd/discovery-concepts"
-        kicker: "Human-centered design guide series"
-  communities:
-    multiple: false
-    community:
-      - link: "/communities/user-experience"
-  top_resources_header: "Design: essential knowledge"
-  top_resources:
-    - title: "An introduction to design"
-      href: "/resources/an-introduction-to-design/"
-      summary: "If your organization needs to ensure compliance with a design standard or align to a brand, a design system can help you achieve those goals more easily than building a site from scratch. Learn how a design system can help you and what you need to know to get started."
-    - title: "An introduction to design systems"
-      href: "/resources/introduction-to-design-systems/"
-      summary: "If your organization needs to ensure compliance with a design standard or align to a brand, a design system can help you achieve those goals more easily than building a site from scratch. Learn how a design system can help you and what you need to know to get started."
-    - title: "USWDS design principles"
-      href: "https://designsystem.digital.gov/design-principles/"
-      summary: "Earn trust by following consistent design principles."
-    - title: "Understanding design in 10 questions"
-      href: "/2023/03/03/understanding-design-in-10-questions/"
-      summary: "Learn how the discipline of design can help your agency improve customer experience."
-    - title: "Bringing design in-house"
-      href: "/2023/01/27/bringing-design-in-house/"
-      summary: "What is design, who are designers, and how can they help your agency? Learn how to build a design team that can help your agency solve “wicked problems” and be more innovative."
-# For more information on managing topics,
-# see https://github.com/GSA/digitalgov.gov/wiki
+# Set the legislation card title and link
+legislation:
+  title: "21st Century Integrated Digital Experience Act"
+  link: "/resources/21st-century-integrated-digital-experience-act/"
+
+# Featured resource to at the top of the page
+featured_resources:
+  resources:
+    - link: "/guides/hcd/discovery-concepts"
+
+# Featured community to display at the top of the page
+featured_communities:
+  - "user-experience"
+
+# Curated list of resources, can be internal or external links
+featured_links:
```

Comment:
changed to `featured_links` because it was clashing with `featured_resources` being used twice.